spec/rails_helper.rb

```diff
@@ -27,13 +28,8 @@
 ActiveRecord::Migration.maintain_test_schema!
 
 RSpec.configure do |config|
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.include ActiveSupport::Testing::Assertions
```

Comment:
I think you're looking for the `change` matcher in RSpec: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/change-matcher.